Sources/Controllers/2.0/OACollapsableNearestPoiWikiView.h

```diff
@@ -9,12 +9,13 @@
 #import "OACollapsableView.h"
 
 @class OAPOI;
+@class OAPOIUIFilter;
 
 @interface OACollapsableNearestPoiWikiView : OACollapsableView
 
 @property (nonatomic, readonly) NSArray<OAPOI *> *nearestItems;
 @property (nonatomic, readonly) BOOL hasItems;
 
-- (void)setData:(NSArray<OAPOI *> *)nearestItems hasItems:(BOOL)hasItems latitude:(double)latitude longitude:(double)longitude;
+- (void)setData:(NSArray<OAPOI *> *)nearestItems hasItems:(BOOL)hasItems latitude:(double)latitude longitude:(double)longitude filter:(OAPOIUIFilter *)filter;
```

Comment:
It would be better to create a second method and pass the default value for filter